README.md

```diff
@@ -51,6 +51,14 @@ ecl-profiles:
      start-year: 2020
      end-year: 2030
      
+  profile_comment_in_stea:
```

Comment:
The new functionality - both fetching profile id from the description and the ability to use non-scalar multipliers is non-trivial and (and slightly non-intuitive as well) and should have a test.

Reply:
fair point. I will look into it.